code/game/machinery/camera/camera.dm

```diff
@@ -69,6 +69,17 @@
 	cancelAlarm()
 	return ..()
 
+/obj/machinery/camera/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>The [src]'s maintenance panel can be <b>screwed [panel_open ? "closed" : "open"]</b>.</span>"
+	if(panel_open)
+		. += "<span class='notice'>Upgrades can be added to the [src] or <b>pried out</b>.</span>"
+		if(!wires.CanDeconstruct())
+			. += "<span class='notice'>The [src]'s <b>internal wires</b> are preventing you from cutting it free.</span>"
+		else
+			. += "<span class='notice'>The [src]'s <i>internal wires</i> are disconnected, but it can be <b>cut free</b>.</span>"
```

Comment:
Sure this wont show `The a security camera`?

Reply:
All these will will show double `the`s:
 
![image](https://user-images.githubusercontent.com/42044220/192159453-1b1bde28-c825-4ed5-bc19-4d2958e76dff.png)

Reply:
👍